continuous_integration/install_wheels.py

```diff
@@ -0,0 +1,52 @@
+"""Install wheels for numpy and scipy.
+
+Without wheels, installation requires doing a build, which is too much.
+The versions of the packages for which wheels are available depends on
+the current Python version.
+
+We use this when building/testing gensim in a CI environment (Travis, AppVeyor,
+etc).
+
+The versions we currently target are:
+
+    - 3.5 (AppVeyor, TravisCI)
+    - 3.6 (AppVeyor, TravisCI)
+    - 3.7 (AppVeyor, TravisCI, CircleCI)
```

Comment:
what's about adding 3.8?

Reply:
Let's deal with that in a separate PR. I just want these builds to start working again. Currently, the failing builds are blocking work from other people from being merged in.